Gemfile

```diff
@@ -164,6 +164,7 @@ group :test do
   gem 'capybara'
   gem 'capybara-email'
   gem 'database_cleaner'
+  gem "webdrivers", '>= 5.3.0'
```

Comment:
can we keep the same quoting and order? also a comment about removing for when merging into the ruby 3 would be useful